packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.ts

```diff
@@ -1,27 +1,113 @@
-import { and, desc, eq } from 'drizzle-orm'
+import { and, desc, eq, getTableColumns, sql, sum } from 'drizzle-orm'
 
 import { Commit } from '../../browser'
 import { database } from '../../client'
 import {
-  createDocumentLogQuery,
-  getCommitFilter,
-} from './_createDocumentLogQuery'
```

Comment:
I 🔥 this archive. Now all the "with metadata" things are in this file. I left a FIXME in evaluations to do the same